core/src/main/java/org/stellar/anchor/filter/CallbackAuthJwtFilter.java

```diff
@@ -0,0 +1,22 @@
+package org.stellar.anchor.filter;
+
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import lombok.NonNull;
+import org.stellar.anchor.auth.ApiAuthJwt;
+import org.stellar.anchor.auth.JwtService;
+
+public class CallbackAuthJwtFilter extends AbstractJwtFilter {
```

Comment:
Can you check if this is used in the platform? If this is used only in the reference server, it should be moved to the reference server project.

Reply:
Moved to reference server.